lib/cli/ui/frame.rb

```diff
@@ -6,6 +7,27 @@ module Frame
       class UnnestedFrameException < StandardError; end
       class << self
         DEFAULT_FRAME_COLOR = CLI::UI.resolve_color(:cyan)
+        DEFAULT_FRAME_STYLE = :box
+        VALID_FRAME_SYLES = [:box, :bracket].freeze
```

Comment:
rather than using symbols directly, my preference would be to:

```ruby
CLI::UI.frame_style = CLI::UI::Frame::BOX_STYLE
```

or something like this, and have that be `:box_style` or whatever under the hood